specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1824,6 +1982,259 @@
         "suppressionDuration"
       ]
     },
+    "AlertRuleTemplatesList": {
+      "description": "List all the alert rule templates.",
+      "properties": {
+        "nextLink": {
+          "readOnly": true,
+          "description": "URL to fetch the next set of alert rule templates.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Array of alert rule templates.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AlertRuleTemplate"
+          }
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "AlertRuleTemplate": {
+      "type": "object",
+      "description": "Alert rule template.",
+      "discriminator": "kind",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "$ref": "#/definitions/AlertRuleKind"
+        }
+      ],
+      "properties": {
+        "etag": {
+          "type": "string",
+          "description": "Etag of the alert rule."
+        }
+      }
+    },
+    "ScheduledAlertRuleTemplate": {
+      "x-ms-discriminator-value": "Scheduled",
+      "type": "object",
+      "description": "Represents scheduled alert rule template.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AlertRuleTemplate"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Scheduled alert rule template properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/BaseAlertRuleTemplateProperties"
```

Comment:
Can you please try to put it on AlertRuleTemplate

Reply:
Cann't be done because both BaseAlertRuleTemplateProperties & extentionProperties Update the same field "properties".

Reply:
I see, that makes sense.